src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -98,8 +99,9 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
+        Remind updatedRemind = personToEdit.getRemind();
```

Comment:
just wondering, do we need an editPersonDescriptor at this stage or later?

Reply:
Since we are probably going to be adding an edit feature in the future we can just leave it in first or add it to v1.2 since the implementation is all done for us already :D